### PR TITLE
Improve markdown's and fix command in e2e Readme.md

### LINKED ISSUE #26: Fix markdowns and typos in command guide in e2e Readme.md
Some markdowns in e2e can be improve and the command ./kishell --configure --server and ./kishell --configure --role have a little mistake, need to be fix for ./kishell configure --server and ./kishell configure --role.

And when i followed this guide i notice a possible Troubleshoot for vm pagination memory in docker environment.

### PR DESCRIPTION
**Pull request by ThiagoTeodoro**
Improve markdown's and fix command in e2e Readme.md

Resolve Issue 26

### CODE REVIEW

File: e2e/README.md [hunk 1]
```diff
@@ -43,4 +66,14 @@ Now it is possible to query the data with a known expected result thanks to the
 
 ```
 ./kishell search --newer="8760h" --query="clientip:172.155.107.128"
-```
\ No newline at end of file
+```
+
+## Troubleshoot
+
+If you have this error when you try to run```docker-compose up --build``` :
```
Comment (robsondepaula) [inline e2e/README.md]: I think we can omit the command in this line and just mention in plain English, like so:

"If you have this error when you try to spin up the stack:"

Comment (ThiagoTeodoro) [reply e2e/README.md]: Thanks for this @robsondepaula.

**Review by robsondepaula — APPROVED** [review]
LGTM
